src/app/webserver/app-pools/app-pools.service.ts

```diff
@@ -118,7 +111,7 @@ export class AppPoolsService {
             //
             // Ping
             if (pool.status == Status.Stopping) {
-                let ob = IntervalObservable.create(1000).subscribe(i => {
+                let ob = interval(1000).subscribe(i => {
```

Comment:
This looks like a legacy code. For safety, please check if we need to call unsubscribe or not.